.github/workflows/check-migrations.yml

```
@@ -0,0 +1,27 @@
+name: Check Migrations
```

To disambiguate between the try-runtime-cli checks maybe
```suggestion
name: Lint Migration Tuples
```